PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -137,13 +137,17 @@ class FemtoDreamTrackSelection : public FemtoDreamObjectSelection<float, femtoDr
   bool isSelectedMinimal(T const& track);
 
   /// Obtain the bit-wise container for the selections
+  /// Pt, eta and dca are not necessarily taken from the track table. For example, for V0 daughters they are recaluated and stored in the V0 table
   /// \todo For the moment, PID is separated from the other selections, hence instead of a single value an std::array of size two is returned
   /// \tparam cutContainerType Data type of the bit-wise container for the selections
   /// \tparam T Data type of the track
   /// \param track Track
+  /// \param Pt pt of the track
+  /// \param Eta eta of the track
+  /// \param Dca dca of the track with respect to primary vertex
   /// \return The bit-wise container for the selections, separately with all selection criteria, and the PID
-  template <typename cutContainerType, typename T>
-  std::array<cutContainerType, 2> getCutContainer(T const& track);
+  template <typename cutContainerType, typename T, typename R>
+  std::array<cutContainerType, 2> getCutContainer(T const& track, R Pt, R Eta, R Dcaxy);
```

Comment:
I would suggest passing instead of the three `Pt`, `Eta`, and `Dcaxy`, one abstract object with a different `typename` and then you extract from it the desired values. As I saw in the other modifications, in few cases that object will be the same `track` but as your comment said it will not be always the case